include/manager.hpp

```diff
@@ -43,67 +43,47 @@ class Manager
     /**
      * @brief Update keyword value.
      *
-     * API can be used to update VPD keyword.
+     * This API is used to update keyword value on the given i_vpdPath and its
+     * redundant path(s) if any taken from system config JSON.
      *
-     * To update a keyword on cache, input path should be D-bus object path.
-     * Eg: ("/xyz/openbmc_project/inventory/system/chassis/motherboard").
+     * To update IPZ type VPD,i_paramsToWriteData should be in the form of
+     * (Record, Keyword, Value). Eg: ("VINI", "SN", {0x01, 0x02, 0x03}).
      *
-     * To update a keyword on hardware, input path should be EEPROM path.
-     * Eg:
-     * ("/sys/bus/i2c/drivers/at24/9-0052/eeprom").
+     * To update Keyword type VPD, i_paramsToWriteData should be in the form of
```

Comment:
nit: instead of i_paramsToWriteData say input parameter for writing should be in the form of ...

Reply:
done